agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -337,8 +337,6 @@ void ap_manager_thread::after_select(bool timeout)
             }
 
             // Set the time for the first radio info polling
-            next_tx_polling_timestamp =
```

Comment:
commit message typo "has been removed fro the ap manager" --> "has been removed **from** the ap manager"

Reply:
I'll fix it on the next rebase. Leaving it unresolved for now so I won't forget.